### DISCUSSION TITLE
Feature Request: LilyGo T-RGB

### DISCUSSION THREAD

**Discussion by PlastiBots** [Bodmer/TFT_eSPI#2245] [New processor/display support or capability request]
Any chance someone can get the LilyGo T-RGB working with this library?  

https://github.com/Xinyuan-LilyGO/T-RGB

**Comment by Bodmer** [comment]
It has an 18 bit parallel interface what would be tricky to suport. The 3 wire SPI mode (no DC pin) is also not supported in this library.

This means support is very unlikely to happen.

**Reply by PlastiBots** [reply]
Ok thanks.

**Reply by electronicsguy** [reply]
https://github.com/crosser/lvgl_esp_lcd
